the_hat_game/players.py

```diff
@@ -83,3 +83,20 @@ def guess(self, words, n_words):
             response_time = 0
             response_code = None
         return {"word_list": word_list, "time": response_time, "code": response_code}
+
+
+class LocalFasttextPlayer(AbstractPlayer):
```

Comment:
Let's discuss here a bit further.
Do you mean moving all contents of `player.py, that is `AbstractPlayer`, `RemotePlayer` and `LocalFasttextPlayer` to `flask_app`?  I think it is a good idea, because I remember when I was playing a game, each time I had to copy a code for player that I tested locally to flask.

I did that, have a look.

Reply:
I made a change moving all contents of `the_hat_game/players.py`  to `flask_app/player.py`. Let's see how it looks.

Reply:
Sorry for a late response here, I meant moving only LocalFasttextPlayer. RemotePlayer is required inside the-hat-game as it it used to run games with remote servers. Another reason to keep it here is that it's expected that participants will change flask-app as they wish, including concrete implementations like DummyPlayer or LocalFasttextPlayer, but they won't change RemotePlayer. They even don't need it in flask-app (if they don't plan to query remote models by themselves there). So I suggest RemotePlayer should stay here.

Reply:
PlayerDefinition and other code should also stay here so similar reasons: flask-app should depend on the-hat-game, not otherwise.

Reply:
ok, done! Now
- `flask_app/player.py` contains `AbstractPlayer`, `LocalDummyPlayer`, `LocalFasttextPlayer`
- `the_hat_game/players.py` contains `PlayerDefinition` + some aux functions + `AbstractPlayer`, `RemotePlayer`.